scss/standalone/patterns_grid-breakout-full.scss

```diff
@@ -0,0 +1,12 @@
+@import '../base';
```

Comment:
Is this file still needed or was it renamed to `layouts_fluid-breakout-full`?

Reply:
no removed